app/policies/organizations/custom_form/question_policy.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class Organizations::QuestionPolicy < ApplicationPolicy
+class Organizations::CustomForm::QuestionPolicy < ApplicationPolicy
```

Comment:
Can we nest this as well for consistency?
```
module Organizations
  module CustomForm
    class QuestionPolicy < ...
```

Reply:
Got missed. Have made the required changes.